README.md

```diff
@@ -36,13 +36,14 @@ dispatch:
     search_order: ['google_ads_source', 'dbt_expectations']
 ```
 
-## Step 2: Install the package
-Include the following google_ads_source package version in your `packages.yml` file.
+## Step 2: Install the package (skip if also using the `google_ads` transformation package)
+If you  are **not** using the [Google Ads transformation package](https://github.com/fivetran/dbt_google_ads), include the following package version in your `packages.yml` file. If you are installing the transform package, the source package is automatically installed as a dependency.
```

Comment:
This reads a bit strange. Can we possibly restructure to be more clear.

Reply:
Note keeping as-is.